### ISSUE TITLE
Text elements need alt text

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#4764]
Some cards might choose to use glyphs or symbols to convey meaning (e.g. `⭐⭐⭐` or `$$`). This unfortunately leads to text that doesn't read properly for users of AT. We need a mechanism by which card authors can supply text to be read by screen readers (could possibly also be purposed to display tooltips).

> Yeah, i agree. We should probably have something tracking figuring out alt-text on Text elements for this kind of scenario. 

_Originally posted by @RebeccaAnne in https://github.com/microsoft/AdaptiveCards/pull/4762#discussion_r486472360_

**Comment by shalinijoshi19** [comment]
@RebeccaAnne, FYI -  backlogged this one.

**Comment by matthidinger** [comment]
Also related, and possible dupe of https://github.com/microsoft/AdaptiveCards/issues/4764
